src/main/java/com/appdynamics/extensions/webspheremq/WMQMonitorTask.java

```diff
@@ -163,11 +162,13 @@ private void extractAndReportMetrics(MQQueueManager mqQueueManager, PCFMessageAg
 				metricsByCommand.get(cmd).put(key, wmqOverride);
 			}
 			if (metricsByCommand.get("MQCMD_INQUIRE_Q_MGR_STATUS") != null) {
+				MetricCreator metricCreator = new MetricCreator(monitorContextConfig, queueManager);
```

Comment:
The astute reader will observe that the `QueueManagerMetricsCollector` was the only implementation that did NOT use the artifact name in its prefix. Why was it special cased? We may never know....but I had to allow null there to keep the behavior unchanged.